PSPrintSDK/OLProductHomeViewController.m

```diff
@@ -33,17 +33,23 @@ -(NSArray *) products{
         _products = [OLKitePrintSDK enabledProducts] ? [OLKitePrintSDK enabledProducts] : [OLProduct products];
         NSMutableArray *mutableProducts = [_products mutableCopy];
         BOOL haveAtLeastOnePoster = NO;
+        BOOL haveAtLeastOneFrame = NO;
         for (OLProduct *product in _products){
             if (!product.labelColor){
                 [mutableProducts removeObject:product];
             }
-            if (product.templateType == kOLTemplateTypePostcard){
+            if (product.productTemplate.templateClass == kOLTemplateClassNA){
                 [mutableProducts removeObject:product];
             }
-            if (product.templateType == kOLTemplateTypeFrame2x2 || product.templateType == kOLTemplateTypeFrame3x3 || product.templateType == kOLTemplateTypeFrame4x4){
-                [mutableProducts removeObject:product];
+            if (product.productTemplate.templateClass == kOLTemplateClassFrame){
+                if (haveAtLeastOneFrame){
```

Comment:
Seeing this does make me think we need to put more thought into the templates JSON response structure. Perhaps a Product like Frames/Posters can have sub variants or something. That way we'd only have a single entry on the print shop screen. Just food for thought :)